### PR TITLE
update code to fix move prometheus-k8s pods issue

### PR DESCRIPTION
**Pull request by liqcui**
update code to fix move prometheus-k8s pods issue.  to get infra_pods, it's already in retry loop, but need to check if no result response, it will assign a default value infra_pods=0, otherwise, it will exist due to **set -o errexit**

### CODE REVIEW

**Comment by liqcui** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.16-nightly-x86-payload-node-density-cni-6nodes

**Comment by qiliRedHat** [comment]
@liqcui LGTM, will leave to @afcollins for review and approval

File: ci-operator/step-registry/openshift-qe/move-pods-infra/openshift-qe-move-pods-infra-commands.sh [hunk 1]
```diff
@@ -69,7 +69,14 @@ function check_monitoring_statefulset_status()
     #wait for 30s to make sure the .status.availableReplicas was updated
     ready_replicas=$( ! oc -n openshift-monitoring get statefulsets $statefulset -oyaml |grep availableReplicas>/dev/null || oc get statefulsets $statefulset -n openshift-monitoring -ojsonpath='{.status.availableReplicas}')
     echo ready_replicas is $ready_replicas
-    infra_pods=$(oc get pods -n openshift-monitoring --no-headers -o wide | grep -E "$infra_nodes" | grep Running | grep "$statefulset" | wc -l  | xargs)
+    #infra_pods=$(oc get pods -n openshift-monitoring --no-headers -o wide | grep -E "$infra_nodes" | grep Running | grep "$statefulset" | wc -l  | xargs)
```
Comment (afcollins) [inline ci-operator/step-registry/openshift-qe/move-pods-infra/openshift-qe-move-pods-infra-commands.sh:72]: Remove this line please, and it's good to merge.

**Review by afcollins — APPROVED** [review]

File: ci-operator/step-registry/openshift-qe/move-pods-infra/openshift-qe-move-pods-infra-commands.sh (hunk 1 above)
Comment (liqcui) [reply ci-operator/step-registry/openshift-qe/move-pods-infra/openshift-qe-move-pods-infra-commands.sh]: removed in new code, thanks!

**Pull request merged**